src/taoensso/sente.cljx

```diff
@@ -928,8 +937,9 @@
               false))))))
 
   (-chsk-connect! [chsk]
-    (when-let [WebSocket (or (enc/oget js/window "WebSocket")
-                             (enc/oget js/window "MozWebSocket"))]
+    (when-let [WebSocket (or (enc/oget goog/global "WebSocket")
+                             (enc/oget goog/global "MozWebSocket")
+                             node-websocket)]
```

Comment:
Is it possible for `goog/global` to have "WebSocket" or "MozWebSocket" when `(= *target* nodejs)`?